src/Microsoft.Extensions.Globalization.CultureInfoCache/CultureInfoCache.cs

```diff
@@ -30,38 +27,18 @@ public static CultureInfo GetCultureInfo(string name, IList<CultureInfo> support
         {
             // Allow only known culture names as this API is called with input from users (HTTP requests) and
             // creating CultureInfo objects is expensive and we don't want it to throw either.
-            if (name == null || supportedCultures == null ||
```

Comment:
Could you modify the `<returns>` on this method to say ...`or <null> if a match` (the if is missing)